.github/workflows/_lint.yml

```
@@ -33,7 +34,8 @@ jobs:
 
     - name: Run npm install
       run: npm install --no-audit --no-fund
-
+    - name: Build  
+      run: npm run build:react
```

This is a redundant and harmful change. Should be reverted. With this change ./packges/jquery project not builds. This is why eslint check fails.
```suggestion
      run: npm run build
```